src/runtime_src/core/tools/common/OptionOptions.cpp

```diff
@@ -48,3 +50,22 @@ OptionOptions::printHelp() const
                                m_positionalOptions, m_globalOptions);
 }
 
+std::vector<std::string> 
+OptionOptions::process_arguments( boost::program_options::variables_map& vm,

```

Comment:
```suggestion
OptionOptions::process_arguments(boost::program_options::variables_map& vm,

```

Reply:
For most file each function has spaces between the parameters and the curly brace. However, I did line up the arguments with the first command. Thanks for pointing this out!